src/common/services/auth/authService.ts

```diff
@@ -0,0 +1,7 @@
+import { TSignupRequest } from './types/requests/TSignupRequest';
+import { TSignupResponse } from './types/responses/TSignupResponse';
+
+export const Ssignup = async (request: TSignupRequest) => {
```

Comment:
```suggestion
export const Ssignup = async () => {
```
Si c'est destiné à être vide, autant ne pas le mettre

Reply:
Même chose ici on sait qu'on auras des trucs a mettre dans TSignupRequest(login,mdp..etc) donc j'ai juste créer le fichier en attendant d'avoir toutes les props nécessaire.